src/index.html

```diff
@@ -7,6 +7,358 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="header__logo-link">
+        <img
+          src="./images/logo.png"
+          alt="MOYO logo"
+          class="header__logo"
+        >
+      </a>
+      <nav class="header__nav">
+        <ul class="header__nav-list">
+          <li class="header__nav-item">
+            <a href="#apple" class="header__nav-link is-active">apple</a>
+          </li>
+
+          <li class="header__nav-item">
+            <a href="#samsung" class="header__nav-link">samsung</a>
+          </li>
+
+          <li class="header__nav-item">
+            <a href="#smartphones" class="header__nav-link">smartphones</a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+             href="#laptops&computers"
+             class="header__nav-link"
+             data-qa="nav-hover"
+            >
+                laptops & computers
+            </a>
```

Comment:
check formatting please
```suggestion
            <a
             href="#laptops&computers"
             class="header__nav-link"
             data-qa="nav-hover"
            >
              laptops & computers
            </a>
```

Reply:
Done